### PR TITLE
New map types

### PR DESCRIPTION
**Pull request by jbwexler**
I made 'Analysis level' not required (blank=True, null=True) and placed it after 'Cognitive Paradigm'. Let me know if you want me to change this.

### CODE REVIEW

**Comment by chrisfilo** [comment]
@ljchang @burnash is this PR satisfying your map type requirements?

**Comment by ljchang** [comment]
@chrisfilo @burnash I think we had discussed possibly having two different beta map types.  Currently, there is a combined ('weight/beta map').  For one of our main use cases (i.e., testing multivariate patterns) we will need to be able to separately filter multivariate weight maps (e.g., our PINES map which will be uploaded soon) from univariate beta maps (e.g., <a href=http://neurovault.org/collections/504/>pain</a> or <a href=http://neurovault.org/collections/504/>emotion</a> datasets).

Any chance we could split beta map into ('multivariate-beta map') and ('univariate-beta map')?

**Comment by chrisfilo** [comment]
Sure. @jbwexler could you rebase and as this?

**Comment by jbwexler** [comment]
sure, no problem

File: neurovault/apps/statmaps/migrations/0047_auto_20150619_0049.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        migrations.AddField(
+            model_name='statisticmap',
+            name='analysis_level',
+            field=models.CharField(choices=[(b'S', b'single-subject'), (b'G', b'group'), (b'M', b'meta-analysis'), (b'Other', b'Other')], max_length=200, blank=True, help_text=b'What level of summary data was used as the input to this analysis?', null=True, verbose_name=b'Analysis level'),
+            preserve_default=True,
+        ),
+        migrations.AlterField(
+            model_name='image',
+            name='polymorphic_ctype',
+            field=models.ForeignKey(related_name='polymorphic_statmaps.image_set+', editable=False, to='contenttypes.ContentType', null=True),
```
Comment (chrisfilo) [inline neurovault/apps/statmaps/migrations/0047_auto_20150619_0049.py]: I wonder where is this one coming from...

**Pull request merged**